localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,186 @@
  innovation_taming_the_skies:0 "Taming The Skies"
  innovation_taming_the_skies_desc:0 "Our tamers and breeders have been hard at work to successfully allow us to use these flying mounts in battle."
 
+ ##############
+ # Tribal Era #
+ ##############
+ 
+ innovation_archmage:0 "Archmages"
+ innovation_archmage_desc:0 "Weaving their intricate enchantments, these mages defend their people with all the magical powers at their disposal. The archmagi brandish magical blades and ancient staves that serve to channel their fierce energies in battle"
+
+ innovation_buccaneer:0 "Buccaneers"
+ innovation_buccaneer_desc:0 "The buccaneers are the unrivaled lords of the seas. They are highly skilled in sea combat, including, but not limited to, firearms and sailing."
```

"Including but not limited to" is an expression usually used in legal work for clarity sake. I suggest changing it to like this:
```suggestion
 innovation_buccaneer_desc:0 "The buccaneers are the unrivaled lords of the seas. They are highly skilled in sea combat. Their expertise in firearms and sailing aggrandize their status as the scourge of the seas."
```

---

Don't tell that to Kel'thuzad or Ner'zhul :D